`task` Logs are failing for multiple resources

For some reason logs are failing when having multiple resources seems that schema.ResourceData is nil 

```sh
panic: interface conversion: interface {} is nil, not *schema.ResourceData
```

--- a/go.mod
+++ b/go.mod
@@ -10,7 +10,7 @@ require (
 	github.com/Azure/go-autorest/autorest/validation v0.3.1 // indirect
 	github.com/alessio/shellescape v1.4.1 // indirect
 	github.com/aohorodnyk/uid v1.1.0
-	github.com/aws/aws-sdk-go v1.40.27
+	github.com/aws/aws-sdk-go v1.40.27 // indirect
 	github.com/aws/aws-sdk-go-v2 v1.11.0
 	github.com/aws/aws-sdk-go-v2/config v1.8.3
 	github.com/aws/aws-sdk-go-v2/service/autoscaling v1.14.0

--- a/iterative/utils/logger.go
+++ b/iterative/utils/logger.go
@@ -1,11 +1,13 @@
 package utils
 
 import (
+	"errors"
 	"fmt"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-	"github.com/sirupsen/logrus"
 	"strings"
 	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/sirupsen/logrus"
 )
 
 var baseTimestamp = time.Now()
@@ -38,6 +40,10 @@ func (f *tpiFormatter) Format(entry *logrus.Entry) ([]byte, error) {
 	data := make(logrus.Fields)
 	for k, v := range entry.Data {
 		data[k] = v
+	}
+
+	if data["d"] == nil {
+		return nil, errors.New("ResourceData is not yet available")
 	}
 
 	d := data["d"].(*schema.ResourceData)

--- a/iterative/utils/logger.go
+++ b/iterative/utils/logger.go
@@ -43,7 +43,7 @@ func (f *tpiFormatter) Format(entry *logrus.Entry) ([]byte, error) {
 	}
 
 	if data["d"] == nil {
-		return nil, errors.New("ResourceData is not yet available")
+		return nil, errors.New("ResourceData is not available")
 	}
 
 	d := data["d"].(*schema.ResourceData)

